scripts/exgfs_wave_post_gridded_sbs.sh

```diff
@@ -252,27 +253,20 @@ source "${USHgfs}/preamble.sh"
     export GRIBDATA=${DATA}/output_$YMDHMS
     export GRDIDATA=${DATA}/output_$YMDHMS
 
-# Gridded data (main part, need to be run side-by-side with forecast
```

Comment:
Can this comment be added back in, or was it removed for a reason?

Reply:
The comment was moved I see not sure why though.

Reply:
Fixed.

Reply:
Thanks @HenryWinterbottom-NOAA !